aspnet/mvc/views/overview.rst

```diff
@@ -0,0 +1,79 @@
+
+View Discovery
+^^^^^^^^^^^^^^
+
+When an action returns a view, a process called *view discovery* takes place. This process determines which view file will be used. There are several scenarios, based on how the view was specified from the action method.
+
+``return View();``
+    The most common approach. In this case the framework will use a view with the same name as the action (with a *.cshtml* extension) located in a folder named the same as the controller. Otherwise, it will look in the *Shared* folder for a view with a name corresponding to the action name.
+
+``return View("List");``
+    Using this syntax, a view with the provided name (and a *.cshtml* extension) is used instead of the current action name. Otherwise, the behavior is as above.
+
+``return View("Views/Home/About.cshtml");``
+    Specify the path from the application root to the view. Using this syntax, the *.cshtml* extension must be specified. The path can optionally start with "/" or "~/".
+
+.. note:: :doc:`Partial views <partial>` and :doc:`view components <view-components>` use similar (but not identical) discovery mechanisms.
+
+.. note:: You can customize the default convention regarding where views are located within the app by using a custom :dn:iface:`~Microsoft.AspNetCore.Mvc.Razor.IViewLocationExpander`.
+
+.. note:: View names may be case sensitive depending on the underlying file system. Case insensitive file systems can specify "about" or "About" or "about.cshtml" or "About.cshtml" to refer to the file *About.cshtml*. However, case-sensitive systems must use case-sensitive mathcing files/folders corresponding to view, action, and controller names.
+
+Passing Data to Views
+^^^^^^^^^^^^^^^^^^^^^
+
+You can pass data to views using several mechanisms. The most robust approach is to specify a *model* type in the view, and then pass an instance of this type to the view from the action. This allows the view to take advantage of strong type checking, and is the recommended approach for most scenarios.
+
+Views can also access ``ViewData``, a dictionary collection that both views and controllers can access and modify. The ``ViewBag`` type provides a dynamic view over the ``ViewData``. Learn more about :doc:`dynamic and strongly typed views <dynamic-vs-static>`.
+  
+Benefits of Using Views
+-----------------------
+
+Views provide `separation of concerns <http://deviq.com/separation-of-concerns/>`_ within an MVC app, encapsulating user interface level markup separately from business logic. ASP.NET MVC views use :doc:`razor syntax <razor>` to make switching between HTML markup and server side logic painless. Common, repetitive aspects of the app's user interface can easily be reused between views using :doc:`layout and shared directives <layout>` or :doc:`partial views <partial>`.
```

Comment:
I think these high level benefits of using views are better discussed at the top of this article right after the "What are Views?" section.